src/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,72 @@
+import { Component } from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import List from '@material-ui/core/List';
+import { withStyles } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+import WebIcon from 'mdi-react/WebIcon';
+import classNames from 'classnames';
+import LanguagePythonIcon from 'mdi-react/LanguagePythonIcon';
+import LanguageCppIcon from 'mdi-react/LanguageCppIcon';
+import LanguageCIcon from 'mdi-react/LanguageCIcon';
+import LanguageJavascriptIcon from 'mdi-react/LanguageJavascriptIcon';
+import LanguageCsharpIcon from 'mdi-react/LanguageCsharpIcon';
+import LanguageCss3Icon from 'mdi-react/LanguageCss3Icon';
+import LanguageSwiftIcon from 'mdi-react/LanguageSwiftIcon';
+import { BUGZILLA_LANGUAGES } from '../../utils/constants';
+
+@withStyles(theme => ({
+  active: {
+    '& $text': {
+      color: theme.palette.primary.main,
+    },
+    '& svg': {
+      fill: theme.palette.primary.main,
+    },
+  },
+  text: {
+    color: theme.palette.grey[800],
+    fontFamily: 'Roboto500',
+  },
+}))
+export default class Sidebar extends Component {
+  render() {
+    const { activeItem, classes } = this.props;
+    const icons = {
+      Python: <LanguagePythonIcon />,
+      JavaScript: <LanguageJavascriptIcon />,
+      Swift: <LanguageSwiftIcon />,
+      C: <LanguageCIcon />,
+      'C++': <LanguageCppIcon />,
+      'C#': <LanguageCsharpIcon />,
+      CSS: <LanguageCss3Icon />,
+    };
+    const items = Object.keys(BUGZILLA_LANGUAGES).map(text => ({
+      text,
+      icon: icons[text],
+    }));
+
+    return (
+      <List disablePadding>
+        {items.map(item => (
+          <ListItem
+            className={classNames({
+              [classes.active]: activeItem === item.text,
+            })}
+            button
+            onClick={this.props.onItemClick}
+            id={item.text}
+            key={item.text}
+            component={Link}
+            to={`/languages/${item.text}`}>
+            <ListItemIcon>{item.icon || <WebIcon />}</ListItemIcon>
+            <ListItemText disableTypography className={classes.text}>
+              {item.text}
+            </ListItemText>
+          </ListItem>
+        ))}
+      </List>
+    );
+  }
+}
```

Comment:
The drawer seems to have a right blue border. Let's remove that.

![screen shot 2018-08-01 at 9 16 44 am](https://user-images.githubusercontent.com/3766511/43523934-dd2eeb72-956b-11e8-8d2f-ae2fd6e4da47.png)

Reply:
When you hover over a project on the far left, the spacing between the scaled project and the drawer should be the same as the distance between neighboring projects.

![screen shot 2018-08-01 at 9 19 19 am](https://user-images.githubusercontent.com/3766511/43524017-22493ec4-956c-11e8-84c3-2ffee8267248.png)